src/e2e-test/java/io/cdap/plugin/pubsub/stepsdesign/PubSubSource.java

```diff
@@ -33,6 +42,32 @@ public void sourceIsPubSub() {
 
   @Then("Open the PubSub source properties")
   public void openThePubSubSourceProperties() {
-    openSourcePluginProperties("GooglePublisher");
+    openSourcePluginProperties("pubsub");
```

Comment:
This change is not required if latest framework steps are used. Please check and revert.

Reply:
Reverted the change